doc-content/enterprise-only/vehicle-route-planning/run-locally-airdistance-proc.adoc

```diff
@@ -0,0 +1,20 @@
+[id='run-locally-airdistance-proc_{context}']
+
+= Run the OptaWeb Vehicle Routing runLocally.sh script in air distance mode
+
+OptaWeb Vehicle Routing can work in air distance mode that calculates travel times based on the distance between two coordinates.
+Use this mode in situations where you need to get OptaWeb Vehicle Routing up and running as quickly as possible and do not want to use an OSM (OpenStreetMap) file.
+Air distance mode is only useful if you need to smoke-test OptaWeb Vehicle Routing and you do not need accurate travel times.
+
+.Prerequisites
+* OptaWeb Vehicle Routing has been successfully built with Maven as described in xref:download-ref-imp-proc_{context}[].
+* Internet access is available.
+
+
+.Procedure
+Run the `runLocally.sh` script with the `--air` argument to start OptaWeb Vehicle Routing in air distance mode:
+
+[source,bash]
+----
+./bin/runLocally.sh --air
```

Comment:
Need to change to 
`./runLocally.sh --air`
(there is no bin folder with that script the only run locally we have is in the source folder)
Maybe add  "Change directory to the rhdm-7.8.0-optaweb-vehicle-routing/sources directory. " as well

Reply:
Updated.